src/test/kotlin/com/github/jensim/megamanipulator/actions/vcs/githubcom/GithubComClientTest.kt

```diff
@@ -12,7 +12,6 @@ import com.github.jensim.megamanipulator.settings.SearchHostSettings.SourceGraph
 import com.github.jensim.megamanipulator.settings.SerializationHolder
 import com.github.jensim.megamanipulator.settings.SettingsFileOperator
 import com.github.jensim.megamanipulator.test.EnvHelper
-import com.github.jensim.megamanipulator.test.EnvHelper.EnvProperty.GITHUB_TOKEN
 import com.github.jensim.megamanipulator.test.EnvHelper.EnvProperty.GITHUB_USERNAME
 import com.github.jensim.megamanipulator.test.TestPasswordOperator
 import com.jetbrains.rd.util.first
```

Comment:
first?
Kotlin func Collection#first()

This might actually be my fault..

Reply:
haha fixed